.mergify.yml

```
@@ -0,0 +1,35 @@
+pull_request_rules:
+  - name: Label the approved PRs
+    conditions:
+      - "#approved-reviews-by>=1"
+    actions:
+      comment:
+        message: "Hello :wave: @{{author}}, The PR is eligible to merge. It will be automerged after 3 working days if not raised any issues."
+      label:
+        add:
+          - automerge
+  - name: Remove the label
+    conditions:
+      - "changes-requested-reviews-by>=1"
```

```suggestion
      - or:
          - "#approved-reviews-by=0"
          - "#changes-requested-reviews-by>=1"
          - "#review-threads-unresolved>=1"
      - label=automerge
```

Expands the conditions that remove a PR from being eligible for automatic merging.